src/ResourceManager/Profile/Commands.Profile.Test/ProfileController.cs

```diff
@@ -37,7 +38,11 @@ public sealed class ProfileController
         private const string SubscriptionIdKey = "SubscriptionId";
 
 
-        public SubscriptionClient SubscriptionClient { get; private set; }
+        public Internal.Subscriptions.SubscriptionClient SubscriptionClient { get; private set; }
+
+        public ResourceManagementClient ResourceManagementClient { get; private set; }
```

Comment:
Do we need all of these resourcemanagemrnt clients?  Seems like we can simplify to a single one.

Reply:
We do need both of them - one is being used in the test code, and one is being used in the cmdlet code.